src/ngraph/node.cpp

```diff
@@ -919,6 +919,13 @@ std::tuple<element::Type, PartialShape>
                     PartialShape::broadcast_merge_into(pshape, get_input_partial_shape(i), autob),
                     "Argument shapes are inconsistent.");
             }
+            else if (autob.m_type == op::AutoBroadcastType::BIDIRECTIONAL)
+            {
+                NODE_VALIDATION_CHECK(
+                    this,
+                    false,
+                    "BIDIRECTIONAL auto broadcast is not supported for elementwise operations");
```

Comment:
Numpy elementwise broadcasting is de facto bidederectional. I am not sure if the exception is needed.
@diyessi What do you think about it?

Reply:
If this check is unnecessary I would vote for removing it.
Treating  BIDIRECTIONAL type as a numpy one in this case should be safe IMO.

Reply:
If numpy and bidirectional are the same, there should only be one.